### ISSUE TITLE
"Root" is not allowed to edit other "Root" users. 

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#2893]
See screenshot: 

![screenshot 2015-02-22 14 46 04](https://cloud.githubusercontent.com/assets/1833361/6318710/b1c0cf30-baa1-11e4-89a5-d635fb367e35.png)

in permissions.yml

```YML
    users:roles-hierarchy:admin: [ root, admin ]
    users:roles-hierarchy:developer: [ root, admin, developer ]
    users:roles-hierarchy:chief-editor: [ root, admin, developer, chief-editor ]
    users:roles-hierarchy:editor: [ root, admin, developer, chief-editor ]
    users:roles-hierarchy:guest: [ root, admin, developer, chief-editor ]
```

**Comment by bobdenotter** [comment]
pinging @MrAdamCook

**Comment by CarsonF** [comment]
This should be fixed in [Permissions](https://github.com/bolt/bolt/blob/master/src/Permissions.php#L184)

**Comment by CarsonF** [comment]
If @bobdenotter is having issues with upgrading to 2.1 we are going to be facing a lot of "issues" from others as well. 

We need to figure out a better solution for these "unversioned" changes.

**Comment by MrAdamCook** [comment]
Looks like you might be missing the following from your permissions file.
users:roles-hierarchy:root: [ root ]

I'll investigate further tonight after work.

**Comment by CarsonF** [comment]
I found the problem.

**Comment by CarsonF** [comment]
This problem should be fixed now. @bobdenotter can you confirm?

**Comment by bobdenotter** [comment]
Fixed by #2931
